static/app/views/releases/detail/overview/releaseAdoption.tsx

```diff
@@ -198,67 +206,96 @@ function ReleaseComparisonChart({
     allowEmptyPeriod: true,
   });
 
+  const adoptionStage = release.adoptionStages?.[project.slug].stage;
+  const adoptionStageLabel = ADOPTION_STAGE_LABELS[adoptionStage];
+  const allEnvironments = environment.length === 0;
+  const multipleEnvironments = environment.length > 1;
+
   return (
-    <RelativeBox>
-      <ChartLabel top="0px">
-        <ChartTitle>
-          {t('Sessions Adopted')}
-          <QuestionTooltip
-            position="top"
-            title={t(
-              'Adoption compares the sessions of a release with the total sessions for this project.'
-            )}
-            size="sm"
+    <Wrapper>
+      <Feature features={['release-adoption-stage']}>
```

Comment:
I've noticed that on releases list and chart marklines we show adoption labels only for mobile projects. We should stay consistent and display it conditionally here too. (you can use `isProjectMobileForReleases` function)